manifests/init.pp

```diff
@@ -77,6 +77,8 @@
   $config_dir                       = $kafka::params::config_dir,
   Stdlib::Absolutepath $bin_dir     = $kafka::params::bin_dir,
   $log_dir                          = $kafka::params::log_dir,
+  $manage_group                     = $kafka::params::manage_group,
```

Comment:
made the requested change.  I'm not clever with git, do I need to make a new PR or does $magic happen to update this one?